pyop2/base.py

```diff
@@ -2786,6 +2747,19 @@ def __iter__(self):
 """
 
 
+class _MapArg(object):
```

Comment:
Since these are immutable, could you have used a [named tuple](https://docs.python.org/2/library/collections.html#collections.namedtuple) instead?